core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -101,7 +102,7 @@ private SqlLibraryOperators() {
    * converts the timezone of {@code datetime} from {@code tz1} to {@code tz2}.
    * This function is only on Redshift, but we list it in PostgreSQL
    * because Redshift does not have its own library. */
-  @LibraryOperator(libraries = {POSTGRESQL})
+  @LibraryOperator(libraries = {POSTGRESQL, REDSHIFT})
```

Comment:
I assume all these REDSHIFT should also move to the other PR.

Reply:
They will for everything except `GREATEST` and `LEAST`, since Redshift should also use the new implementation.

Reply:
This has been cleaned up. No need to specify Redshift here since it will by default get PostgreSQL functions.